WordPress/src/main/java/org/wordpress/android/usecase/social/GetIsSocialSharingLimitedUseCase.kt

```diff
@@ -0,0 +1,16 @@
+package org.wordpress.android.usecase.social
+
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.util.extensions.doesNotContain
+import javax.inject.Inject
+
+class GetIsSocialSharingLimitedUseCase @Inject constructor(
+    private val siteStore: SiteStore,
+) {
+    fun execute(siteId: Long): Boolean =
+        siteStore.getSiteBySiteId(siteId)?.run {
+            !isHostedAtWPCom && planActiveFeaturesList?.doesNotContain(FEATURE_SOCIAL_SHARES_1000) ?: false
```

Comment:
🔍  To make it easier to understand, I would add parenthesis around the second condition, wdyt?

⚠️ Also, I think this fallback is inverted, isn't it? If there's no Features List then I'd argue that site does not contain `FEATURE_SOCIAL_SHARES_1000`, so that should return `true` (similar to [iOS implementation](https://github.com/wordpress-mobile/WordPress-iOS/blob/09143b4ffc4539ead1cbaab944c7af43d5e0c51f/WordPress/Classes/Models/Blog%2BJetpackSocial.swift#L10-L11)), does that make sense?

```suggestion
            !isHostedAtWPCom && (planActiveFeaturesList?.doesNotContain(FEATURE_SOCIAL_SHARES_1000) ?: true)
```

Reply:
Good catch! Makes sense, thanks.